spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -45,15 +45,29 @@ interface FungibleTokenPacketData {
 }
 ```
 
+As tokens are sent across chains using the ICS 20 protocol, they begin to accrue a record of channels for which they have been transferred across. This information is encoded into the `denomination` field. 
+
+The ics20 token denominations are represented the form `{ics20Port}/{ics20Channel}/{denom}`, where `ics20Port` and `ics20Channel` are an ics20 port and channel on the current chain for which the funds exist. The prefixed port and channel pair indicate which channel the funds were previously sent through. If `{denom}` contains `/`, then it must also be in the ics20 form which indicates that this token has a multi-hop record. 
```

Comment:
```suggestion
The ics20 token denominations are represented the form `{ics20Port}/{ics20Channel}/{denom}`, where `ics20Port` and `ics20Channel` are an ics20 port and channel on the current chain for which the funds exist. The prefixed port and channel pair indicate which channel the funds were previously sent through. If `{denom}` contains `/`, then it must also be in the ics20 form which indicates that this token has a multi-hop record - note that this requires that the `/` (slash character) is prohibited in denomination names.
```

Reply:
`prohibited in non-IBC token denomination names`

or something like that. We should be explicit when referring to non-IBC tokens since `denomination` encapsulates both IBC and non-IBC tokens

Reply:
Fixed in 6ea8f4c (both changes including colin's)